spec/system/placements/schools/placements/secondary_school_user_edits_a_placement_spec.rb

```diff
@@ -0,0 +1,404 @@
+require "rails_helper"
+
+RSpec.describe "Secondary school user edits a placement", service: :placements, type: :system do
+  scenario do
+    given_that_placements_exist
+    and_i_am_signed_in
+
+    when_i_am_on_the_organisations_index_page
+    and_i_select_hogwarts
+    then_i_see_the_placements_index_page
+    and_i_see_my_placement
+
+    when_i_click_on_my_placement
+    then_i_see_the_placement_details_page
+
+    when_i_click_on_the_back_link
+    then_i_see_the_placements_index_page
+
+    when_i_click_on_my_placement
+    and_i_click_on_change_expected_date
+    then_i_see_the_when_could_the_placement_take_place_page
+    and_i_see_the_term_dates
+
+    when_i_click_on_the_cancel_link
+    then_i_see_the_placement_details_page
+
+    when_i_click_on_change_expected_date
+    and_i_select_summer_term
+    and_i_click_on_continue
+    then_i_see_the_placement_details_page_with_my_updated_term
+    and_i_see_a_term_updated_success_message
+
+    when_i_click_on_select_a_mentor
+    then_i_see_the_select_a_mentor_page
+    and_i_see_my_mentors
+
+    when_i_click_on_my_mentor_is_not_listed
+    then_i_see_the_mentor_not_listed_details_text
+
+    when_i_click_on_the_cancel_link
+    then_i_see_the_placement_details_page_with_my_updated_term
+
+    when_i_click_on_select_a_mentor
+    and_i_select_john_smith
+    and_i_click_on_continue
+    then_i_see_the_placement_details_page_with_john_smith
+    and_i_see_a_mentor_updated_success_message
+
+    when_i_click_on_change_mentor
+    then_i_see_the_select_a_mentor_page
+    and_i_see_my_mentors
+
+    when_i_select_jane_doe
+    and_i_click_on_continue
```

Comment:
Given you haven't removed `John Smith` it might be worth checking that `John Smith` also appears.

Reply:
Perhaps ignore this given https://github.com/DFE-Digital/itt-mentor-services/pull/1180/files#r1856374691